application/src/main/java/org/togetherjava/tjbot/features/utils/LinkDetections.java

```diff
@@ -0,0 +1,69 @@
+package org.togetherjava.tjbot.features.utils;
+
+import com.linkedin.urls.Url;
+import com.linkedin.urls.detection.UrlDetector;
+import com.linkedin.urls.detection.UrlDetectorOptions;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Utility class to detect links.
+ */
+public class LinkDetections {
+
+    private LinkDetections() {
```

Comment:
any reason to create a seperate class? or it's just git being dumb when you change name of file?

Reply:
yeah there is a reason, because it is used in multiple places (`LinkPreview` and `HelpThreadCreatedListener`)

Reply:
i mean to say the content seems pretty much same as `LinkPreview`, did you just change name of file or added stuff? since it shows all of the code as new in diff.

Reply:
it changed parameter and added one new method